### ISSUE TITLE
Any Plan For CSharp Scripts In Dynamo ? 

### ISSUE THREAD

**Issue by chuongmep** [DynamoDS/Dynamo#12258]
I'm curious, does the team have any plans for bringing C# script same with python script support in Dynamo ?

**Comment by QilongTang** [comment]
@chuongmep Do you mean to support C# coding directly inside Dynamo like how user can leverage Python right now?

**Comment by chuongmep** [comment]
Correct, I think the extension makes it easier for the user to choose. Rhino has adopted this and has delivered high performance.
Something like that : 
C# Scripts : 
```
a =10;
b = 20;
OUT = a+b ;
```
same with python script:
```
a = 10
b = 20
OUT = a+b
```

**Comment by QilongTang** [comment]
@Amoursol @jasonstratton @mjkkirschner I think this fall into the wishlist item so I am moving this there. But it worth discussion for sure.
